jinahub/encoders/image/CLIPImageEncoder/README.md

```diff
@@ -1,13 +1,64 @@
 # CLIPImageEncoder
 
-**CLIPImageEncoder** is a class that wraps the image embedding functionality using the **CLIP** model from huggingface transformers.
+**CLIPImageEncoder** is an image encoder that wraps the image embedding functionality using the **CLIP** model from huggingface transformers.
 
-The **CLIP** model originally was proposed in [Learning Transferable Visual Models From Natural Language Supervision](https://cdn.openai.com/papers/Learning_Transferable_Visual_Models_From_Natural_Language_Supervision.pdf).
+It takes `Document`s with images stored in the `blob` attribute as inputs, and stores the
+resulting image embedding in the `embedding` attribute. You can store original images in
+the `blob` attribute, and they should be in the RGB format and have a shape `[H, W, 3]`). You
+can also choose to pass in already pre-processed images (see the class documentation).

```

Comment:
I don't think we should mention the class documentation if we don't have a link

Reply:
I think this is fine, because the text is meant to be read from Jina Hub, where you have the documentation of the class on the same page. We don't expect people to come to readme hosted on github